shell/platform/android/io/flutter/app/FlutterActivityDelegate.java

```diff
@@ -312,6 +312,12 @@ public void onConfigurationChanged(Configuration newConfig) {}
         if (intent.getBooleanExtra("verbose-logging", false)) {
             args.add("--verbose-logging");
         }
+        // NOTE: all flags provided with this argument are subject to filtering
+        // based on a whitelist in shell/common/switches.cc. If any flag provided
+        // is not present in the whitelist, the engine will immediately shut down.
```

Comment:
"the process will be immediately terminated."

Reply:
Done.